osu.Server.Spectator/Startup.cs

```diff
@@ -63,6 +66,9 @@ public void ConfigureServices(IServiceCollection services)
                 // IdentityModelEventSource.ShowPII = true;
             });
 
+            // Allow a bit of extra time in addition to the graceful shutdown window for asp.net level forced shutdown.
+            services.Configure<HostOptions>(opts => opts.ShutdownTimeout = GracefulShutdownManager.TIME_BEFORE_FORCEFUL_SHUTDOWN.Add(TimeSpan.FromMinutes(1)));
```

Comment:
What is the significance of the added extra minute here? Seems rather insignificant given `TIME_BEFORE_FORCEFUL_SHUTDOWN` is 6 hours...?

Reply:
This is to allow the server to finish the exit process. ie. dispose and clean up entities, remove database multiplayer rooms, etc.

Reply:
I've added an extra note regarding this.